.changeset/selfish-cheetahs-sip.md

```diff
@@ -0,0 +1,35 @@
+---
+'@backstage/plugin-catalog-react': minor
+---
+
+Add an optional `area` parameter to `EntityCard` extensions. A card's area value determines where it should be rendered by the entity content layout, as well as its maximum size.
+
+We are initially supporting only three areas:
+
+- `peek`: used for cards containing infrastucture information (e.g. last builds, deployments, etc.).
+- `info`: used for cards that contain entity metadata (e.g. about, links);
+- `full`: Contains information that plugins add to an entity (e.g. PagerDuty incidents and on-call escalation).
```

Comment:
Names like these are not really areas. They are more variant names, but the variant term is overused, so we encounter conflicts like the one mentioned here:
https://github.com/backstage/backstage/pull/28758/files#r1954130876

Open to suggestions.